### PR TITLE
Update Member Picker Property Editor

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Add Value example

Belongs to: #2714

### CODE REVIEW

**Review by sofietoft — COMMENTED** [review]
Thanks for adding the sample @erikjanwestendorp ! 💪 

One comment:
I found it slightly confusing that the alias is "author", and the variable with the Member ID is also author. Makes it a little hard to distinguish between the two.
Perhaps instead the variable could be something like `authorId`? What do you think? 🤔

**Comment by erikjanwestendorp** [comment]
@sofietoft Good point! 😄 I committed the change!

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Member-Picker/index.md [hunk 1]
```diff
@@ -43,3 +43,48 @@ The member picker opens a panel to pick a specific member from the member sectio
...
+}
+```
+
+If Modelsbuilder is enabled you can get the alias of the desired property without using a magic string:
+
+```csharp
+@{
+
+    // Set the value of the property with alias 'author'
+    content.SetValue(Home.GetModelPropertyType(x => x.Author).Alias, author);
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Member-Picker/index.md:88]:
```suggestion
    content.SetValue(Home.GetModelPropertyType(x => x.Author).Alias, authorId);
```

**Review by sofietoft — COMMENTED** [review]
Thanks for the updates.

One more suggestion, and I think we're good here 💪

**Comment by erikjanwestendorp** [comment]
@sofietoft oops.. my bad. Just committed your suggestion 🙂

**Pull request merged**